### ISSUE TITLE
[Bug] Workspace name is not present anywhere

### ISSUE THREAD

**Issue by mustela** [datumforge/datum-ui#213]
Due to the change mentioned in [this issue](https://github.com/datumforge/datum-ui/issues/204), the workspace name is no longer visible anywhere. This is confusing because I was prompted to set a workspace name, and I can even rename it, but there is now no reference to it.

Additionally, it was difficult to find the page https://console.datum.net/workspace (which, by the way, should be renamed to workspaces if it's meant to display a list of workspaces) in order to create another one. There is currently no easy way to create a new workspace.

**Comment by golanglemonade** [comment]
You can rename it @mustela. It is in workspace settings: https://console.datum.net/workspace-settings/general-settings. Can you please elaborate on what is confusing about not seeing a workspace name when you only have a single one?

Most of the time users will only be creating a workspace at setup, and not actually creating a bunch of additional ones.

**Comment by mustela** [comment]
Sorry, I wasn't asking about renaming it. The issue here is related to UX. After signing up, the first thing the console asks is to name my workspace, but then there's no further reference to it, so there's no context for where I'm working. If it weren't as important as it is, you wouldn't ask for it in the first place; you'd just set a random name.

I would suggest involving a designer to get some feedback, Im not an expert on this area 🙏🏼

**Comment by golanglemonade** [comment]
yep I have a PR linked that will adjust to display the name; we will confirm the dropdown behavior next week and get that fixed up. Appreciate it
